### DISCUSSION TITLE
How to paginate resource collections?

### DISCUSSION THREAD

**Discussion by RezaMortezaee** [spatie/laravel-json-api-paginate#64] [Q&A]
Hi.
Is there any way to paginate resource collections by this package? By passing a paginated resource to the resource collection, paginated links do not show up :(
Thanks

**Comment by cyrildewit** [comment]
Hi @RezaMrtz,

I'm just passing it to the resource class. See an example here: https://github.com/KA-Huis/space-management/blob/acceptance/app/API/V1/Http/Controllers/ReservationController.php

Hopefully this will help you with your issue.

What does your resource class look like. It might filter out the properties from the json paginator instance.
